doc/geany.txt

```diff
@@ -3139,14 +3138,12 @@ The dialog is divided into three sections:
 * Independent build commands (available regardless of filetype).
 * Filetype execute commands.
 
-The filetype and independent sections also each contain a field for the regular
+The filetype and independent build sections also each contain a field for the regular
 expression used for parsing command output for error and warning messages.
 
 The columns in the first three sections allow setting of the label, command,
 and working directory to run the command in.
-
 An item with an empty label will not be shown in the menu.
-
```

Comment:
If merging into a single paragraph, a reflow of the new paragraph might be in order, not a big deal though.

Reply:
I've not done this, ideally yes. I prefer not to reflow too often otherwise the history/diff can introduce more conflicts.

Reply:
Since the `.txt` file is also meant to be read directly (ex. it's installed into docs directory), IMO it's a worthwhile tradeoff to reflow, but again not a big deal.

Reply:
OK, wrapped.